### ISSUE TITLE
Your site could not complete a loopback request

### ISSUE THREAD

**Issue by jamieburchell** [roots/bedrock#574]
* [X] I've read the [guidelines for Contributing to Roots Projects](https://github.com/roots/.github/blob/master/CONTRIBUTING.md)
* [X] This request isn't a duplicate of an existing issue
* [X] I've read the [docs](https://roots.io/documentation/) and followed them (if applicable)
* [X] This is not a personal support request that should be posted on the [Roots Discourse](https://discourse.roots.io/) community

## Description

On a fresh Roots installation on Wordpress 5.6.1 the site health screen shows:

> Your site could not complete a loopback request

Inspecting the nginx log files, I see two requests:

```
example.com 127.0.0.1 - - [04/Feb/2021:20:19:38 +0000] "GET /wp HTTP/1.1" 301 162 "https://example.com/wp" "WordPress/5.6.1; https://example.com" 0.000
example.com 127.0.0.1 - - [04/Feb/2021:20:19:38 +0000] "GET /wp/ HTTP/1.1" 404 12668 "https://example.com/wp/" "WordPress/5.6.1; https://example.com" 0.168
```

Ultimately, the request to `/wp/` results in a 404 and causes the Wordpress message in the site health area.

I checked the Bedrock server config documentation, but I don't see anything specifically about handling the `/wp/` path. I was able to "fix" the issue by adding this to my nginx config:

```
   location ~* /wp/$ {
      return 301 /;
   }
```

I'm not sure if this should be handled by Roots or if this is an issue with Wordpress.

## Steps to reproduce

Visit site health screen after Bedrock installation

**Expected behavior:**

Loopback to work

**Actual behavior:**

Loopback error message

**Reproduces how often:**

100%

## Versions

Bedrock 1.12.8, CentOS 7.9.2009, nginx 1.18.0, PHP 7.4.4, WP 5.6.1

Also confirmed on another nginx and an Apache setup.

**Comment by dalepgrant** [comment]
@jamieburchell Does this help: https://discourse.roots.io/t/wp-sitehealth-curl-error-28/17957

I'm looking for something else but happened to notice your issue looks super similar to that one.

**Comment by jamieburchell** [comment]
Thanks @dalepgrant. In my case I don't think the issues are related since I can see the call being made from the admin to test the loop back, but because it's hitting "/wp" as it's a Bedrock install, it'll 404 there.

**Comment by elvishp2006** [comment]
+1

**Comment by chesio** [comment]
Hi guys,

perhaps you've noticed - this has been fixed in WordPress 5.7: https://core.trac.wordpress.org/ticket/52547
